Typos fix and grammar suggestions (#83)
Consensys/shomei: README.md

### Mode 1: Trace Generation

The first mode is used for the generation of traces so that the prover can retrieve them.

> To use this mode, you need to activate it with these flags: `--enable-trace-generation=true` and `--trace-start-block-number=BLOCK_NUMBER`. The `--trace-start-block-number` flag allows you to define on when we want to start the generation of traces.

Reviewer: "define on when" ? native english speaker here, this is not an improvement.  I think the original text is sufficient but if we are going for grammatical correctness I would suggest 
```suggestion
To use this mode, you need to activate it with these flags: `--enable-trace-generation=true` and `--trace-start-block-number=BLOCK_NUMBER`. The `--trace-start-block-number` flag allows you to define at which block number we want to start the generation of traces.
```

### Mode 2: Serving GetProof for Finalized Blocks

The second mode is used to serve getProof for finalized blocks. This mode does not require the generation of traces, so you should use this `-enable-trace-generation=false` flag. In addition, you should add `--enable-finalized-block-limit=true`, `--use-finalized-block-number=BLOCK_NUMBER`, and `--use-finalized-block-hash=BLOCK_HASH`. 

> Setting these will allow Shomei to stop the sync at this block and thus be able to serve as proof for it. It is recommended to choose a finalized block. Then the node will stop and wait before moving forward. 
> As long as it does not move forward, it can serve as getProof of this block. 

Reviewer: `as` is not correct usage.  The original text, while informal, is more correct.  
```suggestion
Setting these will allow Shomei to stop the sync at this block and thus be able to serve its proof. It is recommended to choose a finalized block. Then the node will stop and wait before moving forward. 
As long as shomai does not move forward, it can serve the getProof method for this block. 
```
